### DISCUSSION TITLE
Adding order number next to sorted column when multiple columns are sorted

### DISCUSSION THREAD

**Discussion by domenicomanna** [KevinVandy/material-react-table#433] [Ideas]
First off, I just want to say that this is an absolutely amazing library! Terrific work.

One feature that I think might be useful is adding in numbers next to sorted columns when multiple columns are sorted. Maybe this is already supported and I just missed it in the docs?  In any case, this will allow the user to easily determine the sort order of the columns instead of needing to remember the order. Checkout https://mui.com/x/react-data-grid/sorting/#multi-sorting, and sort multiple columns to see what I mean. Also see attached screenshot which demonstrates the feature:
![2023-03-17_15 09 31](https://user-images.githubusercontent.com/37960125/225997630-057c72fd-c4d0-4ae5-bc30-0e0ac85378c1.png)

If this is something that you'd like to support, I'd be happy to implement this feature and make a PR for it. Let me know!

**Comment by KevinVandy** [comment]
Good idea. I think I'm going to ship this in the next update

<img width="883" alt="image" src="https://user-images.githubusercontent.com/28243511/226232638-06b71b51-59ec-4260-89b7-206b329b9a16.png">
